### PR TITLE
Fluent-bit helm chart. 

### PR DESCRIPTION
**Pull request by AnirudhPokala**
* Added Fluent-bit helm chart, output to Splunk cloud.

### CODE REVIEW

File: charts/fluent-bit/fluent-bit-splunk/v1.7.8/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+```sh
+To send the logs to Splunk cloud, do the following changes in values.yaml
+
+  [OUTPUT]
+    Name splunk
+    Match *
+    Host XXXXXXXXXXXXXXXXXXX (Hostname of the target Splunk Cloud service)
+    Port 8088
+    Splunk_Token XXXXXXXXXXXXXXXXXXX (Specify the Authentication Token for the HTTP Event Collector interface)
```
Comment (nikhilbandi) [inline charts/fluent-bit/fluent-bit-splunk/v1.7.8/README.md]: Should we also write details of where to get these from in the Splunk Cloud?

File: charts/fluent-bit/fluent-bit-splunk/v1.7.8/values.yaml [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+# kind -- DaemonSet or Deployment
+kind: DaemonSet
+
+# replicaCount -- Only applicable if kind=Deployment
+replicaCount: 1
+
+image:
+  repository: fluent/fluent-bit
+  pullPolicy: Always
```
Comment (nikhilbandi) [inline charts/fluent-bit/fluent-bit-splunk/v1.7.8/values.yaml:11]: Is this _Always_ by default? Any reason for this?

File: charts/fluent-bit/fluent-bit-splunk/v1.7.8/README.md (hunk 1 above)
Comment (AnirudhPokala) [reply charts/fluent-bit/fluent-bit-splunk/v1.7.8/README.md]: For splunk token we can refer to web page which is standard.

File: charts/fluent-bit/fluent-bit-splunk/v1.7.8/values.yaml (hunk 2 above)
Comment (AnirudhPokala) [reply charts/fluent-bit/fluent-bit-splunk/v1.7.8/values.yaml:11]: Yes, the image is being fetched from https://hub.docker.com/r/fluent/fluent-bit/

File: charts/fluent-bit/fluent-bit-splunk/v1.7.8/README.md (hunk 1 above)
Comment (anupbarve) [reply charts/fluent-bit/fluent-bit-splunk/v1.7.8/README.md]: In that case, provide a link to the splunk page then.

Comment (AnirudhPokala) [reply charts/fluent-bit/fluent-bit-splunk/v1.7.8/README.md]: done added.

**Review by anupbarve — APPROVED** [review]

**Pull request merged**